modelforge/potential/utils.py

```diff
@@ -5,6 +5,8 @@
 import torch
 import torch.nn as nn
 from loguru import logger as log
+from modelforge.utils.units import *
```

Comment:
Yeah this is not ideal.  I'm going to switch it up (basically, to be able to convert hartrees to kj/mol I need to add in a different context to openff units, which is just a few lines of code, not a new class.  So I think the easiest is just to set __all__ with a name for this like "chem_context" or something, since I called the context "chem".  That seems to be the simplest way to make sure this is cleaner )